NWPRTXPS/iocBoot/iocNWPRTXPS-IOC-01/st-common.cmd

```diff
@@ -46,6 +49,10 @@ dbLoadRecords("$(MOTOR)/db/motorUtil.db","P=$(MYPVPREFIX)$(IOCNAME):,$(IFIOC)= ,
 
 ## per controller PVs
 dbLoadRecords("$(MOTOR)/db/motorController.db","P=$(MYPVPREFIX),Q=MOT:MTR$(MTRCTRL):")
+#asynSetTraceIOMask("$(XPS_PORT)00", -1, 0x2)
+#asynSetTraceMask("$(XPS_PORT)00", -1, 0x9)
+#asynSetTraceIOMask("$(XPS_PORT)01", -1, 0x2)
+#asynSetTraceMask("$(XPS_PORT)01", -1, 0x9)
```

Comment:
we can readd this very easily if needed - i don't think it should live in source code

Reply:
maybe, but then again they are easy to copy from https://isiscomputinggroup.github.io/ibex_developers_manual/iocs/troubleshooting/ASYN-Trace-Masks-(Debugging-IOC,-ASYN).html#print-commands-and-responses-to-screen